packages/zowe-explorer/i18n/sample/package.i18n.json

```diff
@@ -77,6 +77,7 @@
   "zowe.commands.history": "Toggle if Commands persist locally",
   "zowe.automaticProfileValidation": "Allow automatic validation of profiles.",
   "zowe.settings.version": "Track if migration has been processed",
+  "zowe.security.secureCredentialsEnabled": "Store username and password for z/OS as clear text in Zowe team config profiles if disabled",
```

Comment:
if we are setting autoStore to false during creation the info will not be stored in plain text.

Reply:
I like this, because we still want to provide the more secure solution of getting prompted every time by default. The user needs to explicitly make it less secure by enabling autoStore, which will store in clear text (on the encrypted drive).

Reply:
I agree with the creation setup, this is just a comment about the description of new setting in the Settings.  Could just reword to mention `if autoStore: true credentials will be stored in plain text.`